src/main/java/org/candlepin/subscriptions/db/model/TallySnapshot.java

```diff
@@ -49,6 +55,10 @@
 /**
  * Model object to represent pieces of tally data.
  */
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
```

Comment:
The four Lombok annotations above are the way we should proceed with new models, for sure.  If we can get away with in for existing models, all the better.